README.md

```diff
@@ -168,7 +168,6 @@ You can intermittently poll whether the operation has finished by using the `Pol
   std::string sourceUri = "<a uri to the source blob to copy>";
 
   // Typically, long running operation APIs have names that begin with Start.
```

Comment:
Odd to keep the comment w/o an example.

Reply:
Overaggressive cut and paste (I accidentally put the context example in the LRO example and accidentally removed a line.